app/menu.js

```diff
@@ -162,37 +162,6 @@ const template = [{
   }]
 }]
 
-function addUpdateMenuItems (items, position) {
-  if (process.mas) return
-
-  const version = app.getVersion()
-  const updateItems = [{
-    label: `Version ${version}`,
```

Comment:
I guess we could just remove this line instead of the whole block.
If we do remove the entire block we will not give the user the opportunity to manually check for updates.

Reply:
Those two menus don't work, tested with `v0.6.0`:
![screen shot 2018-06-29 at 12 15 26 pm](https://user-images.githubusercontent.com/26041347/42084413-33bc5514-7b96-11e8-96ed-7ad6dac5ea7b.png)

I do get a popup in the beginning asking me to check github though.
If we want this behavior (manual check) we need to make it work with `electron-updater`

Reply:
ok, then just remove the version and we will deal with it later in another issue (if it will be an option). How does it sound?